skins/oasis/modules/AdController.class.php

```diff
@@ -15,11 +15,6 @@ public function executeConfig() {
 	}
 
 	public function executeTop() {
-		if ( WikiaPageType::isWikiaHub() ) {
-			$leaderboardName = 'HUB_TOP_LEADERBOARD';
-		} else {
-			$leaderboardName = 'TOP_LEADERBOARD';
-		}
-		$this->leaderboardName = $leaderboardName;
+		$this->leaderboardName = 'TOP_LEADERBOARD';
```

Comment:
`$leaderboardName` seems not needed anymore. What do you think about cleaning this code:
https://github.com/Wikia/app/blob/dev/skins/oasis/modules/templates/Ad_Top.php#L6